prediction/src/algorithms/identify/prediction.py

```diff
@@ -221,9 +220,6 @@ def predict_cubes(model_path, patient_id, magnification=1, ext_name=""):  # noqa
                     if cube_mask.sum() < 2000:
                         skipped_count += 1
                     else:
-                        if CROP_SIZE != CUBE_SIZE:
-                            cube_img = helpers.rescale_patient_images2(cube_img, (CUBE_SIZE, CUBE_SIZE, CUBE_SIZE))
```

Comment:
The only case in which rescale_patient_images2 would be executed, which never happens (`CROP_SIZE` is once set to `CUBE_SIZE` and not modified thereafter. Thus, this branch and rescale...2 can be removed.

Reply:
Again, comments here are great but...

Reply:
I'm not completely sure how I should comment in the code base that I removed a method that hasn't been used at all?

Reply:
Well, if there is no need for a branch/code *but someone later might think there should be* that is surely worth a comment around where it "could" be...

Reply:
I added a comment in 8416d58